npm/webpack-dev-server/src/startServer.ts

```diff
@@ -48,13 +49,26 @@ export async function start ({ webpackConfig: userWebpackConfig, template, optio
   }
 
   debug('starting webpack dev server')
+  let webpackDevServerConfig
```

Comment:
Not sure how TypeScript isn't throwing a fit about this. You should see if you can import the correct types for each version of the config, that way the config objects are type checked.

Reply:
The problem is the types are slightly different for WDS v4 and v5 - so either we do it this way, or we need a `// @ts-ignore`.

Reply:
Just union them

Reply:
I will alias the package and try it out. I'll try.

Reply:
Looks like there are [no types for v4](https://www.npmjs.com/package/@types/webpack-dev-server) yet. I just added back in the v3 types.

Reply:
:(

Reply:
I hope the webpack ecosystem can move to TS soon.